commands/utils/packagehandlers/packagehandlers_test.go

```diff
@@ -1,27 +1,376 @@
 package packagehandlers
 
 import (
+	testdatautils "github.com/jfrog/build-info-go/build/testdata"
 	"github.com/jfrog/frogbot/commands/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
 	"github.com/stretchr/testify/assert"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
 	"testing"
 )
 
+type dependencyFixTest struct {
+	fixVersionInfo *utils.FixDetails
+	supportedFix   bool
+}
+
+type pythonIndirectDependencies struct {
+	dependencyFixTest
+	requirementsPath string
+}
+
+const requirementsFile = "oslo.config>=1.12.1,<1.13\noslo.utils<5.0,>=4.0.0\nparamiko==2.7.2\npasslib<=1.7.4\nprance>=0.9.0\nprompt-toolkit~=1.0.15\npyinotify>0.9.6\nPyJWT>1.7.1\nurllib3 > 1.1.9, < 1.5.*"
+
+type pipPackageRegexTest struct {
+	packageName         string
+	expectedRequirement string
+}
+
+var pipPackagesRegexTests = []pipPackageRegexTest{
```

Comment:
Since this collection is used only once, it is best to define it inside the function that uses it (as a local variable),